vultr/packages/vultr/vultr_list.js

```diff
@@ -67,39 +67,47 @@ async function _command(params, commandText, secrets = {}) {
     } catch {
       await install(['@vultr/vultr-node']);
       Vultr = require('@vultr/vultr-node');
-    }   
+    }
 
     const {server} = Vultr.initialize({apiKey: vultrApiKey});
 
     const data = await server.list();
```

Comment:
getting this when I have no server. I think data is an array we may check the length.

![image](https://user-images.githubusercontent.com/35557159/77613560-fb282380-6f50-11ea-9ed8-15db9466dd62.png)

Reply:
change const result and const output to let.

Reply:
> getting this when I have no server. I think data is an array we may check the length.
> 
> ![image](https://user-images.githubusercontent.com/35557159/77613560-fb282380-6f50-11ea-9ed8-15db9466dd62.png)

Fixed it!